Rockerfile

```diff
@@ -3,7 +3,7 @@
 {{ $tag:= (or .Env.DOCKER_TAG "local") }}
 
 FROM golang:1.7-alpine
-RUN apk --update add bash
+RUN apk --update add bash git
```

Comment:
I think it would be better in order to make sure we vendor everything. A vendoring test of sorts